### ISSUE TITLE
md-select label remains highlighted after lost focus

### ISSUE THREAD

**Issue by ganySA** [angular/material#6185]
Hi

For all controls the label element goes grey after lost focus of the input. This behavior is different on the md-select where the label remains the highlighted color after the md-select has lost focus (it does not grey out like on other input controls)

Can be seen on the first demo at:
https://material.angularjs.org/latest/demo/select

When street and city are entered and the controls lose focus, the label is grayed out. The last item state (an md-select) however after selection the label remains blue.

**Comment by DevVersion** [comment]
Fixing the generel label feature depends on #6132 I will create a new PR with other changes today.

**Comment by DevVersion** [comment]
Here is a PR, which fixes that #6207
